Sources/GRToggle/Toggle/GRToggleStyle.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  GRToggleStyle.swift
+//  GoodSwiftUI
+//
+//  Created by Lukas Kubaliak on 27/11/2024.
+//
+
+import SwiftUI
+
+public struct GRToggleStyle: ToggleStyle {
+    
+    // MARK: - Size
+    
+    public enum Size: Int {
+        
+        case small = 24
+        case large = 32
+        
+        var innerCircle: CGFloat { CGFloat(rawValue) / 2.5 }
+        var cornerRadius: CGFloat { 6 }
+        
+    }
+    
+    // MARK: - Alignment
+    
+    public enum Alignment {
+        
+        case leading, trailing
+        
+    }
+    
+    // MARK: - Style
+    
+    public enum Style {
+        
+        case radio, checkBox, checkCircle
+        
+    }
+    
+    // MARK: - Variables
+    
+    var appearance: GRToggleAppearance
+    var style: Style
+    var size: Size
+    var alignment: Alignment
+    
+    @Environment(\.isEnabled) private var isEnabled
+    
+    private var isLarge: Bool { size == .large }
+    private var isCheck: Bool { style == .checkCircle }
+    private var isBox: Bool { style == .checkBox }
+    private var isRadio: Bool { style == .radio }
```

Comment:
I think comparing cases directly would be sufficient. You would have to use `style == .checkbox` instead of `isBox`, but do you really save any code, when you need to define separate computed properties for each case anyway?

Or you can write some code sugar over `Style` enum:
```swift
func is(_ case: Self) -> Bool {
    self == case
}

// usage
if style.is(.checkbox) {
    // ...
}
```